src/App.tsx

```diff
@@ -62,6 +56,11 @@ function App(): ReactElement {
   const [selectedTrack, setSelectedTrack] = useState<Track | null>(null);
   const [currentFrame, setCurrentFrame] = useState<number>(0);
 
+  const [backdropKey, setBackdropKey] = useState<string | null>(null);
```

Comment:
at first glance when reading the code it's hard to tell what this "key" is.  I wonder if `selectedBackdropKey` or just put a comment up here where it's set up, like "the key for the current choice of backdrop images".   It's not necessarily intuitive that there would be multiple possible backdrops.